frontend/src/scenes/insights/insightLogic.tsx

```diff
@@ -445,6 +445,20 @@ export const insightLogic = kea<insightLogicType>({
                         ? urls.insightEdit(short_id, cleanFilters({ ...filters, insight: insightType }, filters))
                         : undefined,
         ],
+        currentFormattedDateRange: [
+            (s) => [s.insight],
+            (insight) => {
+                return dateFilterToText(
```

Comment:
Not critical for this PR but since this sometimes a problematic area for us, calling out some code smell here - in general I wonder if this can be refined to not pass in 5 params and help encapsulate some more of the functionality into the name or helpers.

Reply:
Agreed, it's definitely not the easiest utility function to look at. There's a lot of meaty logic inside of `dateFilterToText` that should be refactored out into `dateFilterToDay` that we should get to in another pr. Thanks for the shout